package-scripts.js

```diff
@@ -0,0 +1,31 @@
+
+const { ifWindows } = require('nps-utils');
+
+/**
+ * Windows: Please do not use trailing comma as windows will fail with token error
+ */
+module.exports = {
+  scripts: {
+    default: 'build',
+    serve: ifWindows(
+      '.\\node_modules\\.bin\\vue-cli-service serve --open',
```

Comment:
You don't need any of this. npm handles windows and linux/mac already.

Reply:
appveyor does not find `./node_modules/.bin`

Reply:
Done